pkg/buildah/login.go

```diff
@@ -59,14 +67,51 @@ func newLoginCommand() *cobra.Command {
 		Use:   "login",
 		Short: "Login to a container registry",
 		Long:  loginDescription,
+
+		PreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			if opts.kubeconfig == "" {
+				return
+			} else {
```

Comment:
This if statement can simply delete `else`, for faster code return and better code Indent .

Reply:
ok...